logging/rust/src/lib.rs

```diff
@@ -0,0 +1,71 @@
+//! logging capability provider
+//!
+//! ```no_compile
+//!   use wasmcloud_interface_logging::{log,debug,info};
+//!   log("info", "log a message at info level");
+//!   info!("this is also at {} level", "info");
+//! ```
+
+mod logging;
+pub use logging::*;
+
+#[cfg(target_arch = "wasm32")]
+pub use wasm_log::log;
+
+#[cfg(target_arch = "wasm32")]
+mod wasm_log {
+    use std::string::ToString;
+    use wasmbus_rpc::RpcResult;
+
+    /// log a text message at a severity level (debug, info, warn, or error)
+    /// parameters may be `&str`, `String`, or `&String`.
+    /// You can also use this crate's macros `debug`,`info`,`warn`, and `error`
```

Comment:
Also just marking this comment if `trace` is added to edit here

Reply:
I used the levels that were implemented in the elixir host.